examples/gradle-v2/versioning-multimodule-example/docs/build.gradle.kts

```diff
@@ -0,0 +1,34 @@
+plugins {
+    kotlin("jvm") apply false
+    `dokka-convention`
+}
+
+dependencies {
+    dokka(project(":childProjectA"))
+    dokka(project(":childProjectB"))
+
+    // No version is necessary, Dokka will add it automatically
+    dokkaHtmlPlugin("org.jetbrains.dokka:versioning-plugin")
+}
+
+dokka {
+    moduleName.set("Dokka Versioning Example")
+}
+
+val currentVersion = "1.0"
+val previousVersionsDirectory: Directory = layout.projectDirectory.dir("previousDocVersions")
+
+dokka {
+    pluginsConfiguration {
+        // Main configuration for the versioning plugin:
+        versioning {
```

Comment:
BTW, what will happen if user will configure `versioning` DSL, but will not add `versioning-plugin` dependency?
Can't we just apply `versioning-plugin` dependency if user configured `versioning` DSL?

Reply:
If versioning is configured, but the plugin isn't provided, then nothing will happen. 

I think we could definitely improve the DSL to automatically add the plugin. I think it'd need to be an explicit `versioning { enabled = true }` option though.

Reply:
I think, that as we provide DSL for it then it will be better to do apply it also via DSL. Could you create an issue for this case?

Reply:
I believe it's also fine, if it will work in the same way as with javadoc.
so, f.e user will need to apply `dokka-versioning` Gradle plugin to get access to `pluginsConfiguration.versioning`.
This will also allow us to understand, how frequently `versioning` plugin is used

Reply:
created an issue for this: https://youtrack.jetbrains.com/issue/KT-72713/DGP-v2-improve-UX-of-configuring-versioning-plugin